mapiproxy/servers/default/emsmdb/emsmdbp_object.c

```diff
@@ -885,7 +892,109 @@ static int emsmdbp_copy_properties(struct emsmdbp_context *emsmdbp_ctx, struct e
 				SRow_addprop(aRow, newValue);
 			}
 		}
-		if (emsmdbp_object_set_properties(emsmdbp_ctx, dest_object, aRow) != MAPISTORE_SUCCESS) {
+
+		ret = emsmdbp_object_set_properties(emsmdbp_ctx, dest_object, aRow);
```

Comment:
If we had to change `emsmdbp_object_set_properties` retval, it would have to be turned to `enum MAPISTATUS`. However enforcing the type would go way beyond the scope of this patch and obfuscate what is achieved here.

Hence, no for this change as part of this PR.